repo2docker/buildpacks/conda/freeze.py

```diff
@@ -2,7 +2,7 @@
 """
 Freeze the conda environment.yml
 
-It runs the freeze in a continuumio/miniconda3 image to ensure portability
+Using conda-lock
```

Comment:
Is `conda-lock` the only dependency? The command `conda-lock --mamba` implies mamba may be needed, though it ran fine for me without.

Reply:
conda-lock can install mamba itself if it's not present